src/pages/[...slug].tsx

```diff
@@ -86,6 +88,7 @@ export const getStaticProps: GetStaticProps<Props, Params> = async (
   const params = context.params!
   const markdown = getContentBySlug(params.slug.join("/"))
   const frontmatter = markdown.frontmatter as Frontmatter
+  const tocItems = markdown.tocItems as Array<ToCItem>
```

Comment:
not needed anymore with the new `PageContent` type
```suggestion
  const frontmatter = markdown.frontmatter
  const tocItems = markdown.tocItems
```

Reply:
Agree... Working on a more robust front matter typing schema where we may need to add something back here, but can remove at the moment.